### ISSUE TITLE
Weird render outcomes but with no errors

### ISSUE THREAD

**Issue by hhr346** [graphdeco-inria/gaussian-splatting#1114]
I followed all the instructions from the repo, and the procedures all seem fine with no errors. But the render outcomes seem weird.
Below is my process of training and rendering. The training loss is around 0.04.
![image](https://github.com/user-attachments/assets/aad62a82-0ac0-4743-81b6-d2bb2d50cb64)
![image](https://github.com/user-attachments/assets/05456a49-80ee-47a7-9089-5aaf67921d00)

Then the comparison of gt and render directory:
![image](https://github.com/user-attachments/assets/380e1c8d-50d8-4c55-a17c-90fe27e26cfe)
![image](https://github.com/user-attachments/assets/0fe986e3-b793-46fc-bfac-e55e827f3447)

The nvcc –version is
nvcc: NVIDIA (R) Cuda compiler driver
Copyright (c) 2005-2021 NVIDIA Corporation
Built on Fri_Dec_17_18:16:03_PST_2021
Cuda compilation tools, release 11.6, V11.6.55
Build cuda_11.6.r11.6/compiler.30794723_0

torch version is:
torch                       1.12.1+cu116
torchaudio                  0.12.1
torchvision                 0.13.1

Is it normal and did I do anything wrong?

**Comment by Wang-Chbo** [comment]
Did you test the previous code, such as these.
Maybe the newest update (Oct 2024 updates) of the repo led to some problems: 
![image](https://github.com/user-attachments/assets/58c2f64c-e971-4638-9ae1-b432d9bb32ff)

**Comment by scott198510** [comment]
@hhr346  I get similar weird render outcomes for colmap data with no errors . I think maybe something is wrong with the latest codes.

![Image](https://github.com/user-attachments/assets/a646b888-69a7-46ef-8f99-5339519d2447)

**Comment by Noya1103** [comment]
I have encountered the same issue. May I ask how you resolved it?
